src/transform.ts

```diff
@@ -16,6 +16,12 @@ export interface Transform {
    * Calculate new field(s) using the provided expresssion(s). Calculation are applied before filter.
    */
   calculate?: Formula[];
+
```

Comment:
Actually, since our master now assume work for 2.0 -- we can actually take this change and get rid of all of them.